src/main/java/com/ridi/books/viewer/reader/pagecontent/DummyPageContent.java

```diff
@@ -21,4 +24,9 @@ public Bitmap renderToBitmap(int bitmapWidth, int bitmapHeight,
             int startX, int startY, int pageWidth, int pageHeight, boolean forHighQuality) {
         return null;
     }
+
+    @Override
+    public List<PageLink> getPageLinkList() {
+        return new ArrayList<>();
```

Comment:
```suggestion
        return Collections.emptyList();
```

(`java.util.Collections` import가 필요합니다)

Reply:
이런함수가 있었군요!
a9bc90e2305f0ca4188acef22b8aea5759108994 에서 적용하였습니다!